.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: generic
+
+sudo: required
+
+services:
+  - docker
+
+env:
+  - KUBERNETES_VERSION=1.9
+  - KUBERNETES_VERSION=1.10
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce wget openssl
+
+script:
+  - make ci-e2e-test KUBERNETES_VERSION=${KUBERNETES_VERSION}
```

Comment:
I'd prefer not to, because:
* I find makefile parameters to be more explicit (it's easier to see what the inputs are)
* envs have to be exported for them to be visible in the makefile